samples/basic/another-remote-module/package.json

```diff
@@ -36,6 +36,8 @@
     "dependencies": {
         "@basic/shared": "workspace:*",
         "@basic/shell": "workspace:*",
+        "@react-aria/toast": "3.0.0-beta.7",
```

Comment:
Why beta versions?

Reply:
I don't know, this is what came up with with the VSCode feature to set a package version. Not a big deal thought as it's a sample.